app/entrypoint.sh

```diff
@@ -110,16 +110,17 @@ function setup {
 
     # copy assets bundles folder into static folder
     local WEBPACK_FILES="./gather/assets/bundles"
-    if [ ${COLLECT_STATIC_FILES_ON_STORAGE:-} ]; then
-        echo "Publish static files..."
-        if [ ${COLLECT_STATIC_FILES_VERSIONED:-} ]; then
+    if [ $COLLECT_STATIC_FILES_ON_STORAGE ]; then
```

Comment:
:nerd_face: 

### set -u
This option causes the bash shell to treat unset variables as an error and exit immediately. Unset variables are a common cause of bugs in shell scripts, so having unset variables cause an immediate exit is often highly desirable behavior.

Reply:
The way to skip the error is what we have before :grimacing:

Reply:
and we use the `-u` option to check that there are not missing variables...